apps/condo/domains/common/components/TicketCard/TicketOverview.tsx

```diff
@@ -37,7 +37,7 @@ export const TicketOverview: React.FC<TTicket> = ({
                     </Typography.Link>
                 </Link>
             </Col>
-            <Space direction={'vertical'} size={4}>
+            <Space direction={'vertical'} size={4} style={{ width: '100%' }}>
```

Comment:
max-width?

Reply:
it's no matter for this case, since it's just a container for text and will act exactly the same with "max-width" specified